### DISCUSSION TITLE
Regarding the eval command

### DISCUSSION THREAD

**Discussion by razetime** [tomtheisen/stax#46] [General]
eval says that it takes a string, and evaluates it as data to be pushed to the stack. However, it doesn't seem to evaluate blocks when put in string. 

Will this be a future feature?

**Comment by tomtheisen** [comment]
`e` only "eval"s strings using the semantics of the implicit input parsing.  That does not include blocks.

However, the forthcoming `l` instruction should take care of this use case.  `l` on an array/string will execute it as a string of stax code.  Therefor `"{}"l` will leave an empty block on the stack.

1. Push a string containing open and close curly braces.
2. `eval` the 2-character string.  The string is popped, and the code `{}` pushes an empty block to the stack.
3. The stack contents is now a single block with no contents.

This will probably be out in 1.2.
